### PR TITLE
Enhanced Toggle Visibility button in scene tree editor

### PR DESCRIPTION
**Pull request by sersoong**
Toggle Visibility button can control multiple items.
test

### CODE REVIEW

File: editor/scene_tree_editor.cpp [hunk 1]
```diff
@@ -68,24 +68,18 @@ void SceneTreeEditor::_cell_button_pressed(Object *p_item, int p_column, int p_i
...
+		_toggle_visible(n);
+		List<Node *> selection = editor_selection->get_selected_node_list();
+		if (selection.size() > 1) {
+			for (List<Node *>::Element *E = selection.front(); E; E = E->next()) {
+				Node *nv = E->get();
+				ERR_FAIL_COND(!nv);
+				if (nv == n) {
+					continue;
+				}
+				_toggle_visible(nv);
```
Comment (akien-mga) [inline editor/scene_tree_editor.cpp]: The problem with this approach is that you're creating an UndoRedo action for each step, so if you toggle visibility for 10 nodes and want to undo it, you need to press Undo 10 times.

Comment (sersoong) [reply editor/scene_tree_editor.cpp]: Ok,I get it.
I will fix it soon.

**Comment by sersoong** [comment]
Fixed,but 2.1 is not done yet.

**Comment by akien-mga** [comment]
Thanks a lot for your contribution!
We are now entering a strict release freeze for Godot 3.0 and will only consider major bug fixes. We won't merge new features and enhancements anymore until 3.0 is released.

Moving this PR to the 3.1 milestone, to be reviewed once the release freeze is lifted. It could eventually be cherry-picked for a future 3.0.1 maintenance release if it doesn't change the user-facing APIs and doesn't compromise the engine's stability.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**